scripts/ccpp_capgen.py

```diff
@@ -309,6 +309,17 @@ def compare_fheader_to_mheader(meta_header, fort_header, logger):
         if not list_match:
             errmsg = 'Variable mismatch in {}, variables missing from {}.'
             errors_found = add_error(errors_found, errmsg.format(title, etype))
+            if etype == "metadata header":
```

Comment:
Can't add a comment to the spot I'd like to, but can we remove the comment on line 268?

> The headers should have the same variables in the same order